docs/DEBUGGING.md

```diff
@@ -1,26 +1,60 @@
 ## Xdebug
 
-To use Xdebug in the `frontend` and/or `admin` app, it must be installed on the client image. To install, you will need to either manually pass `REQUIRE_XDEBUG_FRONTEND=1` as an env var when bringing up the app or run the make command `up-app-xdebug-frontend`
+To use Xdebug in the `frontend` and/or `admin` app, it must be installed on the client image. To install, you will need
+to either manually pass `REQUIRE_XDEBUG_FRONTEND=1` as an env var when bringing up the app or run the make
+command `up-app-xdebug-frontend`
 
-To install Xdebug on the API, either manually pass `REQUIRE_XDEBUG_API=1` as an env var when bringing up the app or run the make command `up-app-xdebug-api`.
+To install Xdebug on the API, either manually pass `REQUIRE_XDEBUG_API=1` as an env var when bringing up the app or run
+the make command `up-app-xdebug-api`.
 
 You can confirm installation by running `php -v` in the container and seeing that it reports the Xdebug version.
 
-Since v3 performance with Xdebug enabled is only slightly slower than without so leaving Xdebug enabled shouldn't be a huge performance hit compared with previous versions.
+Since v3 performance with Xdebug enabled is only slightly slower than without so leaving Xdebug enabled shouldn't be a
+huge performance hit compared with previous versions.
 
 ## PHPStorm PHPUnit XDebug setup
 
-To enable step through xdebug in PHPStorm/IntelliJ you'll need to ensure the setup for PHP and PHPUnit are pointing to the relevant Docker PHP install.
+To enable step through xdebug in PHPStorm/IntelliJ you'll need to ensure the setup for PHP and PHPUnit are pointing to
+the relevant Docker PHP install.
+
+## Create frontend and api servers
+
+Go to `Preferences` and select `Servers` under the PHP dropdown. When setting up each server (frontend and api) you will
+need to add and click the following:
+Add a `name` for each server and a `host name` - api (for api) and digideps.local (for frontend)
+The `Port` needs to be 443 for both servers Select tick box `Use path mappings` and find folder (client or api) in
+the `File/Directory` expand the folder, click the pencil icon, and type the following `/var/www` then click apply
 
 ### Create a Docker PHP CLI Interpreter
-The steps below are for setting this up for the API container. This will need to be followed again for setting up Client/Admin with any api specific namings or file locations updated to match the client directory.
 
-Go to Settings > Languages & Frameworks > PHP > Click `...` next to CLI Interpreter.
+The steps below are for setting this up for the API container. This will need to be followed again for setting up
+Client/Admin with any api specific namings or file locations updated to match the client directory.
```

Comment:
```suggestion
The steps below are for setting this up for the API container. This will need to be followed again for setting up Client/Admin with any api specific namings or file locations updated to match the client directory.
```